homeassistant/components/niko_home_control/const.py

```diff
@@ -4,3 +4,18 @@
 
 DOMAIN = "niko_home_control"
 _LOGGER = logging.getLogger(__name__)
+
+
+NIKO_HOME_CONTROL_THERMOSTAT_MODES_MAP = {
+    "off": 3,
+    "cool": 4,
+    "auto": 5,
+}
+
+
+class NIKO_HOME_CONTROL_THERMOSTAT_MODES(int):
```

Comment:
This class shouldn't be uppercased, also use IntEnum. But this should live in the library

Reply:
Ok done, but not sure if this one is for the lib, it contains HA modes that niko doesn't have, it's a map from ha to niko modes/presets.